CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ else()
 
     if (DEFINED SERIAL_FLASHER_INTERFACE_UART)
         list(APPEND srcs
+            src/esp_stubs.c
```

Comment:
If I am not mistaken, we also need to add the stub source to the sources list when using `ESP_PLATFORM`.

Reply:
Also, looks like this might be a source of errors in the future, I will explore refactoring the project CMakeLists

Reply:
You're right, I've added the file to sources when using ESP_PLATFORM

https://github.com/higaski/esp-serial-flasher/commit/c4e4a7c6491a2cf7068f160bcc5da2ddd6868721